.github/workflows/swarmctl-smoke.yml

```
@@ -11,4 +11,17 @@ jobs:
+          if [ -d "repos/packages/agent-os/src" ]; then
+            echo "SWARMCTL_ENABLED=1" >> "$GITHUB_ENV"
+            echo "AGENT_OS_SRC=$PWD/repos/packages/agent-os/src" >> "$GITHUB_ENV"
+          else
+            echo "SWARMCTL_ENABLED=0" >> "$GITHUB_ENV"
+          fi
+      - name: Run swarmctl smoke
+        if: env.SWARMCTL_ENABLED == '1'
+        run: python3 scripts/swarmctl.py smoke
+      - name: Skip note
+        if: env.SWARMCTL_ENABLED != '1'
+        run: echo "Skipping swarmctl smoke: repos/packages/agent-os/src is not present in this checkout."
```

actionlint (1.7.11)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)